karabo_data/geometry2/__init__.py

```diff
@@ -359,10 +366,20 @@ def plot_data_fast(self, data, axis_units='px', frontview=True):
         frontview : bool
           If True (the default), x increases to the left, as if you were looking
           along the beam. False gives a 'looking into the beam' view.
+        ax : `~matplotlib.axes.Axes` object, optional
+          Axes that will be used to draw the image (default: None).
```

Comment:
Nitpick: the default is to create a new figure. `None` is an implementation detail.